FCChhAnalyses/FCChh/tth_mumu/analysis.py

```diff
@@ -15,8 +15,8 @@
 # pre-produced input files
 comp = cfg.Component(
     'example',
-     files = ["root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_ll012j_5f/events100.root"]
-     files = ["root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_tth01j_5f_hmumu/events0.root"]
+     files = ["root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_ll012j_5f/events100.root"],
+     #files = ["root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_tth01j_5f_hmumu/events0.root"]
```

Comment:
@clementhelsens this needs review, the current code does not work as it is, so we could exclude one of the two files or do the following:

```python
     files = ["root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_ll012j_5f/events100.root",
              "root://eospublic.cern.ch///eos/experiment/fcc/hh/generation/DelphesEvents/fcc_v01/pp_tth01j_5f_hmumu/events0.root"
               ]
```

Reply:
Hi Javier, your proposed change is fine. `comp` is a way to manually test the analysis with a small / local sample. In fact maybe you can comment l.36 where it says 
```
# uncomment to try with local file 
selectedComponents = [comp]
```